backend/development.ini.sample

```diff
@@ -331,18 +331,23 @@ website.base_url = %(basic_setup.website_base_url)s
 ; webdav.ui.enabled = True
 
 ####
-# EMAIL-COMMON
+# JOB EXECUTION
 ####
 
-### Email sending configuration ###
+### configuration of potentially lengthy jobs ###
 # processing_mode may be sync or async,
-# if async is choosen, you need also to run mail_notifier daemon
-# see README for more info.
-; email.processing_mode = sync
+# if async is choosen, you need also to run:
+#  - a custom rq worker (see backend/README.md)
+#  - a redis server
+; jobs.processing_mode = sync
```

Comment:
This is a breaking change, should be notified in https://github.com/tracim/tracim/issues/2785.

Reply:
Done in https://github.com/tracim/tracim/issues/2785#issuecomment-637544988